src/Image/Format.php

```diff
@@ -152,11 +152,11 @@ protected static function create($formatID)
     {
         switch ($formatID) {
             case static::ID_JPEG:
-                return new static($formatID, 'jpg', 'image/jpeg', array('jpg', 'pjpeg', 'jfif'));
+                return new static($formatID, 'image/jpeg', 'jpg', array('jpg', 'pjpeg', 'jfif'));
             case static::ID_WBMP:
-                return new static($formatID, 'jpg', 'vnd.wap.wbmp');
+                return new static($formatID, 'vnd.wap.wbmp', 'jpg');
```

Comment:
Shouldn’t the extension be `wbmp` here?
And the correct mime type starts with `image/` I think.
```suggestion
                return new static($formatID, 'image/vnd.wap.wbmp', $formatID);
```

Reply:
Yes, I guess it should. I have just swapped the arguments order. Committed now.